server/src/main/java/org/apache/iotdb/db/conf/IoTDBConstant.java

```diff
@@ -97,6 +97,8 @@ private IoTDBConstant() {
   public static final String SCHEMA_FOLDER_NAME = "schema";
   public static final String SYNC_FOLDER_NAME = "sync";
   public static final String QUERY_FOLDER_NAME = "query";
+  public static final String TRACING_FOLDER_NAME = "tracing";
```

Comment:
not used

Reply:
It's used in IoTDBDescriptor `conf.setTracingDir(FilePathUtils.regularizePath(conf.getBaseDir() + IoTDBConstant.TRACING_FOLDER_NAME));`